.travis/main.sh

```diff
@@ -29,9 +29,9 @@ setup_dependencies() {
   echo "INFO:  Setting up dependencies."
   echo "#########################################################"
 
-  sudo apt update -y
-  sudo apt install realpath python python-pip -y
-  sudo apt install --only-upgrade docker-ce -y
+  sudo apt-get update -y
```

Comment:
sudo apt is the new one. is there a reason why we should take the old apt-get?

Reply:
No Reason. In RC.3 we want to make it consistent.

Reply:
MDD-56